info.yml

```diff
@@ -114,23 +114,17 @@ configuration:
           type: dict
         default_value:
         - caption: Assets
-          entity_type: Asset
+          entity_type: Task
```

Comment:
please add a comment here to point out that for backwards comp reasons, the default value isn't using the most optimal setup etc (and maybe what that would look like). Some kind of breadcrumb or little heads up just to remind users i think would be useful.